site/content/entry/yuzu-progress-report-apr-2022/index.md

```diff
@@ -0,0 +1,332 @@
++++
+date = "2022-05-07T12:00:00-03:00"
+title = "Progress Report April 2022"
+author = "GoldenX86"
+coauthor = "CaptV0rt3x"
+forum = 0
++++
+
+Hello yuz-ers! April has been kind to us, providing CPU and Kernel performance improvements, several GPU emulation changes, UI tweaks and more!
+
+<!--more--> 
+
+## Saving Princess Peach yet again
+
+Continuing his work on the GameCube/Wii [Hagi](https://en.wikipedia.org/wiki/Japanese_destroyer_Hagi_(1920)) and Nintendo 64 [Hovercraft](https://en.wikipedia.org/wiki/Landing_Craft_Air_Cushion) [official Nintendo emulators support](https://yuzu-emu.org/entry/yuzu-progress-report-mar-2022/#the-vulkan-emulator) (totally not in violation of their own EULA), [byte[]](https://github.com/liamwhite) has introduced several new PRs to further improve the emulation status of the titles included within `Super Mario 3D All-Stars`.
+
+First things first, support for GLSL in `Super Mario Sunshine` is needed too, not everyone can run Vulkan. 
+This is achieved by adding {{< gh-hovercard "8133" "support for indirect addressing" >}} in OpenGL.
+
+This change doesn’t include support for GLASM at the moment, devs aren't too fond of having to deal with assembly shaders code. 
+Anyone would get angry if you get told to fix an issue in a car engine, and the only given tools for the job are a rock and a stick.
```

Comment:
```suggestion
This change doesn’t include support for GLASM at the moment, since developers aren't too fond of having to deal with Nvidia assembly shader code.
Imagine being asked to fix an issue in a car engine, and the only given tools for the job are a rock and a stick.
```